### PR TITLE
Add composer Web UI URL preview and auto bucket discovery

### PR DESCRIPTION
**Pull request by mik-laj**
Hello,

I often lose the address to the web interface of Google Composer. The application should show this address if I need it. I know that this is not a good time to come up with new features, but these small improvements makes the experience much more pleasant.

In addition, I introduced bucket name detection. The user does not have to provide the bucket name. The name of the composer's environment is sufficient.

Thanks in advance,

### CODE REVIEW

File: oozie-to-airflow/run-sys-test [hunk 1]
```diff
@@ -361,6 +365,15 @@ function prepare_configuration {
...
 }
 
+function prepare_gcloud_configuration {
+    if [[ -z "${COMPOSER_DAG_BUCKET}" ]]; then
+        COMPOSER_DAG_BUCKET=$(gcloud beta composer environments describe "${COMPOSER_NAME}" --location "${COMPOSER_LOCATION}" --format='value(config.dagGcsPrefix)')
+        COMPOSER_DAG_BUCKET=${COMPOSER_DAG_BUCKET%/dags}
+        COMPOSER_DAG_BUCKET=${COMPOSER_DAG_BUCKET#gs://}
+    fi
+    COMPOSER_WEB_UI_URL=$(gcloud beta composer environments describe "${COMPOSER_NAME}" --location "${COMPOSER_LOCATION}" --format='value(config.airflowUri)')
+}
```
Comment (potiuk) [inline oozie-to-airflow/run-sys-test]: Please add echos that show the environment variables. I think it is super-important to show those derived values for any kind of debugging or diagnostics.

File: oozie-to-airflow/run-sys-test [hunk 2]
```diff
@@ -361,6 +365,15 @@ function prepare_configuration {
     fi
 }
 
+function prepare_gcloud_configuration {
```
Comment (potiuk) [inline oozie-to-airflow/run-sys-test]: Similarly as in other functions - please add header and footer showing what's going on. Again for diagnostics it is very important for the user to see what's going on.

Comment (mik-laj) [reply oozie-to-airflow/run-sys-test]: I updated a PR.

File: oozie-to-airflow/run-sys-test (hunk 1 above)
Comment (mik-laj) [reply oozie-to-airflow/run-sys-test]: I updated a PR.

**Comment by potiuk** [comment]
Nice!

**Comment by mik-laj** [comment]
@potiuk Can I merge? Does anyone want to do a review?  This is important for me because it will improve my current job.

**Comment by potiuk** [comment]
Go ahead. Merge it

**Pull request merged**